widget/playground/src/components/Slider/Slider.tsx

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers */
```

Comment:
Can you remove this comment and define some constants?

Reply:
fixed